eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -519,20 +519,20 @@ object Commitments {
 
   def revocationHash(seed: ByteVector32, index: Long): ByteVector32 = Crypto.sha256(revocationPreimage(seed, index))
 
-  def sendCommit(commitments: Commitments, keyManager: KeyManager)(implicit log: LoggingAdapter): Try[(Commitments, CommitSig)] = {
+  def sendCommit(commitments: Commitments, channelKeyManager: ChannelKeyManager)(implicit log: LoggingAdapter): Try[(Commitments, CommitSig)] = {
```

Comment:
Same here (and everywhere in the PR). I think we should keep the name of these variables `keyManager`, as long as the type is correct we don't need the variable name change. It also removes noise from the PR.

Reply:
Done